kratos/containers/container_variable_data/container_variable_data.cpp

```diff
@@ -0,0 +1,279 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+#include <sstream>
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "utilities/parallel_utilities.h"
+#include "containers/container_variable_data/expressions/literal/literal_expression.h"
+#include "containers/container_variable_data/expressions/literal/literal_flat_expression.h"
+
+// Include base h
+#include "container_variable_data.h"
+
+namespace Kratos {
+
+template <class TContainerType>
+ContainerVariableData<TContainerType>::ContainerVariableData(ModelPart& rModelPart)
+    : mpModelPart(&rModelPart)
+{
+}
```

Comment:
Wrong initialization order!

```suggestion
template <class TContainerType>
ContainerVariableData<TContainerType>::ContainerVariableData(ModelPart& rModelPart)
    : mpExpression(),
      mpModelPart(&rModelPart)
{
}
```

Reply:
done :)